README.md

```diff
@@ -99,3 +105,22 @@ Generate a static build under `frontend/dist` directory:
 ```bash
 yarn workspace frontend generate
 ```
+
+### Database migrations
+
+[db-migrate](https://github.com/db-migrate/node-db-migrate) tool used to apply data migrations. More details you can find here: https://db-migrate.readthedocs.io/en/latest/
+
+Install
+```bash
+npm install -g db-migrate
+```
+
+Change directory
+```bash
+cd db
+```
+
+Create migration
+```bash
+db-migrate create MIGRATION_NAME -e local
+```
```

Comment:
@upalinski 
Please, add what should be in `up` and `down` files.

Reply:
added. please, check.